src/main/java/frc/robot/commands/TeleopSwerve.java

```diff
@@ -71,6 +77,32 @@ protected double calculateRotationVal() {
     return rotationLimiter.calculate(rotationVal);
   }
 
+  // protected double calculateRotationVal() {
```

Comment:
If this method is not useful for later, suggest to delete it.

Reply:
This was an attempted implementation by Khalil and Fernando which wasn't working so we had to abandon it for new market. It should ideally be implemented for north bay though so I think leaving it in is fine.

It should use the gyro to keep the heading constant whenever the driver doesn't want the robot to rotate.

Reply:
Is this feature similar to the one we removed, which is discretize?

Reply:
No discretize tries to solve a similar but different problem.

Discretize tries to solve the issue where when driving forwards and rotating, the robot drives in an arc.

But this is trying to solve the issue where due to many different factors, the rotation can slowly move over time so it uses pid to maintain the current heading.